docs/src/topics/flavors/vpcless.md

```diff
@@ -27,3 +30,17 @@ supported in this flavor and it uses VXLAN for pod to pod communication.
     ```bash
     kubectl apply -f test-cluster.yaml
     ```
+
+### Flatcar
+1. Before management cluster initialization, you need to run `export EXP_KUBEADM_BOOTSTRAP_FORMAT_IGNITION=true`
+2. A Flatcar image must be imported with `cloud-init` feature (see: https://www.flatcar.org/docs/latest/installing/community-platforms/akamai/#importing-an-image)
```

Comment:
it looks like these instructions end with an `IMAGE_ID` env variable could we add a step here calling out something like `export FLATCAR_IMAGE_NAME =$IMAGE_ID` or something like that?